projection/channel/view/channels.go

```diff
@@ -0,0 +1,354 @@
+package view
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/crypto-com/chain-indexing/appinterface/pagination"
+	"github.com/crypto-com/chain-indexing/appinterface/projection/view"
+	"github.com/crypto-com/chain-indexing/internal/utctime"
+	jsoniter "github.com/json-iterator/go"
+
+	sq "github.com/Masterminds/squirrel"
+	"github.com/crypto-com/chain-indexing/appinterface/rdb"
+	_ "github.com/crypto-com/chain-indexing/test/factory"
```

Comment:
Any reason we need this test factory?

Reply:
Copy and paste from `projection/account/view/accounts.go` 😂 
Let me remove this! Thanks for pointing that out!